jlm/tooling/CommandLine.hpp

```diff
@@ -177,6 +177,7 @@ class JlmOptCommandLineOptions final : public CommandLineOptions
   struct StatisticsCommandLineArgument
   {
     inline static const char * Aggregation_ = "print-aggregation-time";
+    inline static const char * AndersenAnalysis_ = "print-andersen-analysis";
```

Comment:
Let's just all it "AndersenAliasAnalysis" or "AndersenAnalysis". All these command line arguments need overhauling to something more sane.

Reply:
I'm sorry, I don't quite understand what you are requesting here. Do you want me to remove the trailing underscore? I can add the word "Alias" of course, but your comment makes it seem like that is not what you are after?

Reply:
The command line flag is currently called "print-andersen-analysis". I would like to move away from this unhandy schema, and rather just call it by its name: "AndersenAliasAnalysis" or "AndersenAnalysis":

```
inline static const char * AndersenAnalysis_ = "AndersenAliasAnalysis";
```

Reply:
But then the command line flag for enabling statistics collection would be `--AndersenAliasAnalysis`, which looks a bit confusing, and is different from all other statistics enabling flags?

Is it not better left for another PR that:
 - changes the statistics enabling flags in a consistent manner, possibly moving to a
`--statistics=Comma,Separated,List` format?
 - unifies the list between `JlmOptCommand` and `JlcCommand`, avoiding all the duplication between them?

Reply:
You are absolutely right. I created an issue for it: #357 . The list between jlc and jlm-opt should already be unified and in sync (TM).